Source/Extensions/ArrayExtenstions.swift

```diff
@@ -0,0 +1,35 @@
+//
```

Comment:
We've got a typo in a file name
ArrayExtenstions.swift -> ArrayExtensions.swift